public/static/js/task.js

```diff
@@ -261,12 +261,61 @@ mciModule.controller('TaskHistoryDrawerCtrl', function($scope, $window, $locatio
 
       });
 
-      mciModule.controller('TaskCtrl', function($scope, $rootScope, $now, $timeout, $interval, md5, $filter, $window, $http, $locationHash) {
+      mciModule.controller('TaskCtrl', function($scope, $rootScope, $now, $timeout, $interval, md5, $filter, $window, $http, $locationHash, $mdDialog, mciSubscriptionsService, notificationService, $mdToast) {
         $scope.userTz = $window.userTz;
         $scope.haveUser = $window.have_user;
         $scope.taskHost = $window.taskHost;
         $scope.jiraHost = $window.jiraHost;
+        $scope.subscriptions = [];
+        $scope.triggers = [
+            // TODO uncomment as we implement these
+            // {
```

Comment:
Please add two more blocks here, one for outcome, one for succcess.

Reply:
I believe we discussed not needing 3 types since they are all just outcome. Let's just change this once we implement it